test_typing_inspect.py

```diff
@@ -44,27 +74,35 @@ def test_generic(self):
         T = TypeVar('T')
         samples = [Generic, Generic[T], Iterable[int], Mapping,
                    MutableMapping[T, List[int]], Sequence[Union[str, bytes]]]
-        nonsamples = [int, Union[int, str], Union[int, T], ClassVar[List[int]],
-                      Callable[..., T], ClassVar, Optional, bytes, list]
+        nonsamples = [int, Union[int, str], Union[int, T]]
+        nonsamples += CLASSVAR_GENERIC
+        nonsamples += [Callable[..., T], Optional, bytes, list]
         self.sample_test(is_generic_type, samples, nonsamples)
 
     def test_callable(self):
         samples = [Callable, Callable[..., int],
                    Callable[[int, int], Iterable[str]]]
-        nonsamples = [int, type, 42, [], List[int],
-                      Union[callable, Callable[..., int]]]
-        self.sample_test(is_callable_type, samples, nonsamples)
-        class MyClass(Callable[[int], int]):
+        nonsamples = [int, type, 42, [], List[int]]
+        try:
+            nonsamples.append(Union[callable, Callable[..., int]])
+        except TypeError:
+            # python 3.5.3 TypeError: Union[arg, ...]:
+            # each arg must be a type. Got <built-in function callable>.
```

Comment:
Use `sys.version_info` here.

Reply:
ok. I even created a global variable like the others